### PR TITLE
Add AssetID() func to Coin

### PR DESCRIPTION
**Pull request by ivan-bozhytksyi**

### CODE REVIEW

**Review by vikmeup — APPROVED** [review]

File: coin/coins.go [hunk 1]
```diff
@@ -20,10 +25,25 @@ type Coin struct {
...
+
 func (c *Coin) String() string {
 	return fmt.Sprintf("[%s] %s (#%d)", c.Symbol, c.Name, c.ID)
 }
 
+func (c Coin) AssetID() AssetID {
+	return AssetID(coinPrefix + fmt.Sprint(c.ID))
+}
+
+func (c Coin) TokenAssetID(t string) AssetID {
```
Comment (vikmeup) [inline coin/coins.go:38]: nice!

File: coin/coins.go [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-// 2021-06-18 08:30:36.901143 -0700 PDT m=&#43;0.001565402
+// 2021-08-17 09:18:06.774336 &#43;0300 EEST m=&#43;0.002442799
 // using data from coins.yml
 package coin
 
 import (
 	"fmt"
 )
 
+const (
```
Comment (vikmeup) [inline coin/coins.go:11]: can we move prefix into `asset` lib?

Comment (ivan-bozhytksyi) [reply coin/coins.go:11]: You mean asset package? 
There are prefixes already, but I had to duplicate them here to avoid circular dependencies.

Comment (vikmeup) [reply coin/coins.go:11]: oh I see, that make sense then

**Pull request merged**